src/main/scala/collections/Steps.scala

```diff
@@ -0,0 +1,12 @@
+package collections
+
+class Steps {
+  val steps = List(2, 3, 5)
+
+  def countSteps(): LazyList[Int] = 1 #:: count(LazyList(1))
+
+  private def count(l: LazyList[Int] = LazyList.empty): LazyList[Int] = {
```

Comment:
much better. The only thing, there is no need to make a Steps class, could be an object

Reply:
made Steps an object